controllers/providers/aws/constructors.go

```diff
@@ -40,20 +38,22 @@ func (w *AwsWorker) GetAutoScalingBasicBlockDevice(name, volType, snapshot strin
 	if encrypt != nil {
 		device.Ebs.Encrypted = encrypt
 	}
-	if iops != 0 && strings.EqualFold(volType, "io1") {
+	if iops != 0 && common.ContainsEqualFold(AllowedVolumeTypesWithProvisionedIOPS, volType) {
 		device.Ebs.Iops = aws.Int64(iops)
 	}
+	if throughput != 0 && common.ContainsEqualFold(AllowedVolumeTypesWithProvisionedThroughput, volType) {
```

Comment:
Is there a default on the SDK side if it's set to 0?

Reply:
old SDK did not even have the option to configure Throughput, So I have to bump up to the new version, Tested `0` on gp3 for ProvisionedThroughput it was treated as `nil`, and the default value is set in both the launch template and launch configuration

Reply:
👌🏻